releasenotes/notes/pulse-gate-calibrations-78fd3fa5a5328761.yaml

```diff
@@ -42,3 +42,10 @@ features:
 
     Circuits with pulse gates may still be lowered to the Pulse Schedule representation by using
     the :py:mod:`qiskit.scheduler`.
+
+    The calibrated gate can also be now be transpiled using the regular transpilation process::
+
+        transpiled_circuit = transpile(circ, backend)
+
+    The transpiled circuit will have the transpiled gates on the same qubit as the original
+    circuit and will not be unrolled to the basis gates.
```

Comment:
```suggestion
    circuit and will not unroll them to the basis gates.
```

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/5080/commits/78bdbb09195b2df1927d2150981c806862c57ec3.